tool/tsh/tsh.go

```diff
@@ -4113,3 +4126,20 @@ func forEachProfile(cf *CLIConf, fn func(tc *client.TeleportClient, profile *cli
 
 	return trace.NewAggregate(errors...)
 }
+
+func parseTTL(rawTTL string) (time.Duration, error) {
+	if rawTTL == "" {
+		return 0, nil
+	}
+
+	ttl, err := time.ParseDuration(rawTTL)
+	if err != nil {
+		d, err := strconv.ParseInt(rawTTL, 10, 0)
+		if err != nil {
+			return 0, trace.BadParameter("failed to parse TTL value: %v", err)
```

Comment:
I would include the actual value that we failed to parse in the error message.
```suggestion
			return 0, trace.BadParameter("failed to parse TTL value %q: %v", rawTTL, err)
```